README.md

```diff
@@ -31,6 +32,12 @@ $ npm install -g bower
 $ npm install -g grunt-cli
 ```
 
+* Sass - You're going to use Sass[Sass](http://sass-lang.com/) to compile CSS durring your grunt task. Make sure you have ruby and then install Sass using gem install:
```

Comment:
@tatethurston Sass[Sass] shows up as a duplicate. Remove the Sass from before the [].